outdated/mywarnings.py

```diff
@@ -27,7 +23,3 @@ class OutdatedCacheFailedWarning(OutdatedWarningBase):
     Warning emitted when writing to or reading from the cache
     fails with an exception.
     """
-
-
-if os.environ.get('OUTDATED_IGNORE'):
-    filterwarnings("ignore", category=OutdatedWarningBase)
```

Comment:
This should not be removed, as using `check_outdated` directly instead of `warn_if_outdated` can still create warnings sometimes, and it's still good to be able to disable that with OUTDATED_IGNORE.

Reply:
Ah yes make sense. I'll get back to this MR in a few days. I would also like to add tests.